iroh/src/magicsock/node_map/path_state.rs

```diff
@@ -6,7 +6,7 @@ use std::{
     time::{Duration, Instant},
 };
 
-use iroh_base::key::NodeId;
+use iroh_base::NodeId;
```

Comment:
I find it a little odd that the imports don't come from `crate::NodeId`... but I don't think this is a problem.  Probably rust-analzyer will keep inserting this anyway.